app/poros/user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr_reader :id, :name, :image, :friends, :games
+  def initialize(data)
+    @id = data[:attributes]
+    @name = data[:attributes]
+    @image = data[:attributes]
+    @friends = []
+  end
+
```

Comment:
The add_friends method can be called in the initialization rather than in the facade